examples/cross_build/toolchain_packages/toolchain_macos_linux_cross/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+from conan import ConanFile
+from conan.tools.files import get, copy, download
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+
+class ArmToolchainPackage(ConanFile):
+    name = "arm-toolchain"
```

Comment:
need to fix the name as is this is the same as the other example - but we are not only targetting ARM

`linux-gnu-macos-toolchain` or `linux-gnu-macos-cross-toolchain` or something like that may work better :/